README.md

```
@@ -1,5 +1,10 @@
 # INT Smart Triage AI 2.0 🎯
 
+[![Tests](https://img.shields.io/badge/tests-16%2F16%20passing-brightgreen)]()
```

markdownlint-cli2 (0.18.1)

No empty links

(MD042, no-empty-links)